dask/dataframe/tests/test_io.py

```diff
@@ -630,16 +633,20 @@ def test_to_castra():
     finally:
         c.drop()
 
-    dsk, keys = a.to_castra(compute=False)
-    assert isinstance(dsk, dict)
-    assert isinstance(keys, list)
-    c, last = keys
-    assert last[1] == a.npartitions - 1
+    delayed = a.to_castra(compute=False)
+    assert isinstance(delayed.dask, dict)
+    c, _ = delayed.compute()
```

Comment:
I fixed it and i also added a test for your code snippet, but can you please make sure the is a proper fix?